internal/pkg/ai-functions/handler/handler_ai_function_create_testcase.go

```diff
@@ -147,7 +150,41 @@ func processSingleTestCase(ctx context.Context, factory functions.FunctionFactor
 			TestCaseID:      aiCreateTestCaseResponse.TestCaseID,
 		})
 	} else {
-		result, err := aiHandlerUtils.GetChatMessageFunctionCallArguments(ctx, factory, req, openaiURL, tp.Prompt, systemPrompt, callbackInput)
+
+		var (
+			f          = factory(ctx, "", req.GetBackground())
+			systemMsg0 = openai.ChatCompletionMessage{
+				Role:    openai.ChatMessageRoleSystem,
+				Content: "background: Your name is Erda AI Assistant. You are trained by Erda.",
```

Comment:
no need this message, it's auto-added by ai-proxy.

Reply:
OK